helm/fiftyone-teams-app/README.md

```diff
@@ -0,0 +1,725 @@
+This key is used to encrypt storage credentials in the MongoDB database.
+
+The generate an `encryptionKey`, run this Python code
+
+```python
+from cryptography.fernet import Fernet
+print(Fernet.generate_key().decode())
+```
+
+Voxel51 does not have access to this encryption key and cannot reproduce it.
+Please store this key in a safe place.
+If the key is lost, you will need to
+
+1. Schedule an outage window
+    1. Drop the storage credentials collection
+    1. Replace the encryption key
+    1. Add the storage credentials via the UI again.
+
+Storage credentials no longer need to be mounted into
+pods with appropriate environment variables being set.
+Users with `Admin` permissions may use the FiftyOne Teams UI to manage storage
+credentials by navigating to `https://<DEPOY_URL>/settings/cloud_storage_credentials`.
+
+FiftyOne Teams continues to support the use of environment variables to set
+storage credentials in the application context and is providing an alternate
+configuration path for future functionality.
+
+### Proxies
+
+FiftyOne Teams supports routing traffic through proxy servers.
+To configure this, set the following environment variables on
+
+1. All pods, in the environment (`*.env`):
+
+    ```yaml
+    http_proxy: http://proxy.yourcompany.tld:3128
+    https_proxy: https://proxy.yourcompany.tld:3128
+    no_proxy: <apiSettings.service.name>, <appSettings.service.name>, <teamsAppSettings.service.name>
+    HTTP_PROXY: http://proxy.yourcompany.tld:3128
+    HTTPS_PROXY: https://proxy.yourcompany.tld:3128
+    NO_PROXY: <apiSettings.service.name>, <appSettings.service.name>, <teamsAppSettings.service.name>
+    ```
+
+1. The pod based on the `fiftyone-teams-app` image (`teamsAppSettings.env`)
+
+    ```yaml
+    GLOBAL_AGENT_HTTP_PROXY: http://proxy.yourcompany.tld:3128
+    GLOBAL_AGENT_HTTPS_PROXY: https://proxy.yourconpay.tld:3128
+    GLOBAL_AGENT_NO_PROXY: <apiSettings.service.name>, <appSettings.service.name>, <teamsAppSettings.service.name>
+    ```
+
+The `NO_PROXY` and `GLOBAL_AGENT_NO_PROXY` values must include the Kubernetes
+service names that may communicate without going through a proxy server.
+By default, these service names are
+
+- `teams-api`
+- `teams-app`
+- `fiftyone-app`
+
+If the service names were overridden in `*.service.name`, use these values instead.
+
+By default, the Global Agent Proxy will log all outbound connections
+and identify which connections are routed through the proxy.
+To reduce the logging verbosity, add this environment variable to your `teamsAppSettings.env`
+
+```ini
+ROARR_LOG: false
+```
+
+#### Text Similarity
+
+Since version v1.2, FiftyOne Teams supports using text similarity
+searches for images that are indexed with a model that
+[supports text queries](https://docs.voxel51.com/user_guide/brain.html#brain-similarity-text).
+To use this feature, use a container image containing
+`torch` (PyTorch), instead of the `fiftyone-app` image.
```

Comment:
```suggestion
```

Reply:
This statement is confusing to me, because it fails to note that you need all the same packages as `fiftyone-app` _and_ PyTorch...  I say drop it and just use the sentence below.